cypress/e2e/talks-interaction.cy.ts

```diff
@@ -0,0 +1,238 @@
+
+    cy.get('#language-filters > [data-filter="*"]').click;
+    cy.get('[data-filter="docker"]').click();
+    cy.get('[data-filter="english"]').click();
+    cy.get('#language-filters > [data-filter="*"]').should(
+      "not.have.class",
+      "active"
+    );
+    cy.get('[data-filter="english"]').should("have.class", "active");
+    cy.get('[data-filter="polish"]').should("not.have.class", "active");
+    cy.get("@git-pl").should("not.be.visible");
+    cy.get("@steem-pl").should("not.be.visible");
+    cy.get("@bitcoins-en").should("not.be.visible");
+    cy.get("@blockchain2-pl").should("not.be.visible");
+    cy.get("@blockchain1-pl").should("not.be.visible");
+    cy.get("@lightning-en").should("not.be.visible");
+    cy.get("@docker-pl").should("not.be.visible");
+
+    cy.get('[data-filter="polish"]').click();
+    cy.get('#language-filters > [data-filter="*"]').should(
+      "not.have.class",
+      "active"
+    );
+    cy.get('[data-filter="english"]').should("not.have.class", "active");
+    cy.get('[data-filter="polish"]').should("have.class", "active");
+    cy.get("@git-pl").should("not.be.visible");
+    cy.get("@steem-pl").should("not.be.visible");
+    cy.get("@bitcoins-en").should("not.be.visible");
+    cy.get("@blockchain2-pl").should("not.be.visible");
+    cy.get("@blockchain1-pl").should("not.be.visible");
+    cy.get("@lightning-en").should("not.be.visible");
+    cy.get("@docker-pl").should("be.visible");
+
+    cy.get('#language-filters > [data-filter="*"]').click;
+    cy.get('[data-filter="blockchain"]').click();
+    cy.get('[data-filter="english"]').click();
+    cy.get('#language-filters > [data-filter="*"]').should(
+      "not.have.class",
+      "active"
+    );
+    cy.get('[data-filter="english"]').should("have.class", "active");
+    cy.get('[data-filter="polish"]').should("not.have.class", "active");
+    cy.get("@git-pl").should("not.be.visible");
+    cy.get("@steem-pl").should("not.be.visible");
+    cy.get("@bitcoins-en").should("be.visible");
+    cy.get("@blockchain2-pl").should("not.be.visible");
+    cy.get("@blockchain1-pl").should("not.be.visible");
+    cy.get("@lightning-en").should("be.visible");
+    cy.get("@docker-pl").should("not.be.visible");
+
+    cy.get('[data-filter="polish"]').click();
+    cy.get('#language-filters > [data-filter="*"]').should(
+      "not.have.class",
+      "active"
+    );
+    cy.get('[data-filter="english"]').should("not.have.class", "active");
+    cy.get('[data-filter="polish"]').should("have.class", "active");
+    cy.get("@git-pl").should("not.be.visible");
+    cy.get("@steem-pl").should("be.visible");
+    cy.get("@bitcoins-en").should("not.be.visible");
+    cy.get("@blockchain2-pl").should("be.visible");
+    cy.get("@blockchain1-pl").should("be.visible");
+    cy.get("@lightning-en").should("not.be.visible");
+    cy.get("@docker-pl").should("not.be.visible");
+  });
+
+  afterEach(() => {
+    cy.clearCookies();
```

Comment:
`cy.clearCookies();` is unnecessary. Cypress clear session after each "it" automatically